src/Native/Runtime/PalRedhawk.h

```diff
@@ -367,7 +367,7 @@ typedef struct _CONTEXT {
     UIntNative GetIp() { return Eip; }
     UIntNative GetSp() { return Esp; }
 } CONTEXT, *PCONTEXT;
-#include "poppack.h"
+#include "coreclr/poppack.h"
```

Comment:
Please do not modify the includes of poppack.h / pshpackXXX.h, but rather add the path where they are located to the include paths in CMakeLists.txt.